tests/data/pronounce/litexa/main.litexa

```diff
@@ -11,39 +11,39 @@ launch
 
 TEST "say yes"
   launch
-  alexa: launch, /I really like my cat/i
+  alexa: launch, /I really like my cat/
   user: AMAZON.YesIntent
   alexa: null, /my cat/
-  alexa: null, /phoneme/
+  alexa: null, /phoneme.*hidden/i
 
 TEST "say no"
   launch
-  alexa: launch, /I really like my cat/i
+  alexa: launch, /I really like my cat/
   user: AMAZON.NoIntent
-  alexa: null, /goodbye then/i
+  alexa: null, /goodbye then/
 
 TEST "say yes en-CA"
   setRegion en-CA
   launch
-  alexa: launch, /I really like my ferret/i
+  alexa: launch, /I really like my ferret/
   user: AMAZON.YesIntent
   alexa: null, /my ferret/
-  alexa: null, /phoneme/
+  alexa: null, /phoneme.*concealed/i
```

Comment:
Could combine these into the same regex check. Something like: `/my ferret.*<phoneme[^>]*>concealed`/